src/Makefile.am

```diff
@@ -764,6 +768,102 @@ bitcoin_util_LDADD = \
 bitcoin_util_LDADD += $(BOOST_LIBS)
 #
 
+# bitcoin-chainstate binary #
+#
+# TODO: Remove blockfilter.cpp and index/blockfilterindex.cpp after merge of
+#       "Improve Indices on pruned nodes via prune blockers" #21726
```

Comment:
nit: Maybe remove this TODO and either create a review comment (after merge) on 21726, or create a tracking comment in the parent b-cs issue?

Reply:
Comment removed as of 9050346f04d0435ad71d93ba1df4f5c126c0337b
Added to "Project-wide TODOs": https://github.com/bitcoin/bitcoin/issues/24303